providers/gupshup/package.json

```
@@ -0,0 +1,84 @@
+{
+  "name": "@novu/gupshup",
+  "version": "",
```

```suggestion
  "version": "0.5.1",
```

I think that was causing some of the build errors aswell :)